pkgs/by-name/wa/waydroid-script/package.nix

```
@@ -0,0 +1,57 @@
+{
+  python3,
+  waydroid,
+
+  fetchFromGitHub,
+  stdenv,
+  lib,
+  makeWrapper,
+}:
+stdenv.mkDerivation rec {
+  name = "waydroid-script";
```

```suggestion
  pname = "waydroid-script";
```
`name` is an implicit attr, normally constructed as `"${pname}-${version}"`, do not set it explicitly